test/NumSharp.UnitTest/README.md

```diff
@@ -7,4 +7,52 @@ Install `Codecov` from https://www.nuget.org/packages/Codecov.
 Add `.nuget\packages\codecov\1.1.0\tools` to system bin Path
 
 `PS D:\Projects\NumSharp> OpenCover.Console -register:user -target:"dotnet.exe" -targetargs:test -filter:"+[NumSharp.UnitTest]*" -output:"coverage.xml" -oldstyle`
-`PS D:\Projects\NumSharp> codecov -f coverage.xml -t <codecov-numsharp-token>`
\ No newline at end of file
+`PS D:\Projects\NumSharp> codecov -f coverage.xml -t <codecov-numsharp-token>`
+
+
+### NDArray generation for Unit Test cases
+
+Generating NDArray constants by hand is tedious and error prone, so here
+is some Python code that spits out C# definitions for static NDArrays
+that can be used in unit tests:
+
+```python
```

Comment:
The generating code should be placed in `NumSharp.UnitTest/Utilities/ndarray-generator.py` (or named anything of that sort)